File: 8-kyu/rock-paper-scissors!/python/solution.py
def rps(p1, p2):
    if p1 == p2:
        return 'Draw!'
    if (p1 == 'rock' and p2 == 'scissors') or (p1 == 'paper' and p2 == 'rock') or \
        (p1 == 'scissors' and p2 == 'paper'):
        return 'Player 1 won!'
    return 'Player 2 won!'

File: 8-kyu/rock-paper-scissors!/javascript/solution.js
const rps = (p1, p2) => {
  const losers = { scissors: "rock", rock: "paper", paper: "scissors" };
  if (losers[p1] === p2) {
    return "Player 2 won!";
  } else if (losers[p2] === p1) {
    return "Player 1 won!";
  }
  return "Draw!";
};